policy-report/pkg/api/wgpolicyk8s.io/v1beta1/policyreport_types.go

```diff
@@ -133,6 +135,13 @@ type PolicyReport struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
+	// Source is an identifier for the policy engine that manages this report
+	// Use this field if all the results are produced by a single policy engine.
+	// If multiple policy engines are producing results in a single PolicyReport,
```

Comment:
If the results are produced by multiple sources e.g. different engines or scanners, use...